ibm/mas_devops/playbooks/oneclick_core.yml

```diff
@@ -34,25 +34,13 @@
         fail_msg: "One or more required environment variables are not defined"
 
     - name: Check for required environment variables for UDS
-      when: (bas_provider =='UDS')
       assert:
         that:
           - lookup('env', 'UDS_CONTACT_EMAIL') != ""
           - lookup('env', 'UDS_CONTACT_FIRSTNAME') != ""
           - lookup('env', 'UDS_CONTACT_LASTNAME') != ""
         fail_msg: "One or more required environment variables are not defined"
 
-
-    - name: Check for required environment variables for DRO
-      when: (bas_provider =='DRO')
-      assert:
-        that:
-          - lookup('env', 'DRO_CONTACT_EMAIL') != ""
-          - lookup('env', 'DRO_CONTACT_FIRSTNAME') != ""
-          - lookup('env', 'DRO_CONTACT_LASTNAME') != ""
-        fail_msg: "One or more required environment variables are not defined"
```

Comment:
Can we put this back but update the check so it checks for both i.e.

```
lookup('env', 'DRO_CONTACT_EMAIL') != "" or lookup('env', 'UDS_CONTACT_EMAIL') != ""
```
note: I haven't tested this line so it might need tweaking to get it working